### DISCUSSION TITLE
Slack?

### DISCUSSION THREAD

**Discussion by revans611** [PrismLibrary/Prism#2466] [General] [answered]
I am just starting a new WPF Prism type project after a long absence.  Is there no longer a PrismLibrary channel on Slack?

**Comment by brianlagunas** [comment, marked as answer]
No, but if you sponsor either [myself](https://github.com/sponsors/brianlagunas) or [@dansiegel](https://github.com/sponsors/dansiegel) at the $10 tier or higher, you can gain access to our private Discord
